packages/components/src/components/card/card.tsx

```diff
@@ -30,8 +30,19 @@ export class card {
   render() {
     return (
       <div class={this.getCssClassMap()}>
+        <div class="card__header">
+        <h4 class="card-title">Testtitel Header</h4>
+        <h5 class="card-subtitle">Subtitel Header</h5>
+          <slot name="header" />
+        </div> 
+        <img class="card-img-top" src="http://placehold.it/400x300" alt="Card image"></img>
         <div class="card__body">
           <slot/>
+          <t-button variant="primary" class="hydrated">Click me!</t-button>
```

Comment:
no need to embed `t-button`, it should come in a slot (probably will be injected as part of of the default slot)